beartype/_util/py/utilpyinterpreter.py

```diff
@@ -104,7 +104,7 @@ def is_python_optimized() -> bool:
         try:
             PYTHONOPTIMIZE_int = int(PYTHONOPTIMIZE_str)
         # If doing so raises *ANY* exception whatsoever, return false.
-        except:
+        except (ValueError, TypeError):
```

Comment:
For efficiency reasons, it'd be useful to either:

* **Globalize** the tuple `(ValueError, TypeError)` as a global variable and then reuse that variable here (rather than reify this tuple each call to this frequently called tester function). That said...
* **Just catch `except Exception:`** seems like the more general-purpose approach. It's probably best not to assume the exact types of exceptions raised by builtin magic like the `int()` constructor.

Otherwise, all of this seems awesome! Team Beartype! *Raaaaaw!*